pkgs/development/haskell-modules/configuration-common.nix

```
@@ -1408,6 +1408,15 @@ self: super: {
       # 2021-04-09: test failure
       # PR pending https://github.com/expipiplus1/update-nix-fetchgit/pull/60
       doCheck = false;
+
+      patches = [
+        # 2021-05-17 compile with hnix >= 0.13
+        # https://github.com/expipiplus1/update-nix-fetchgit/pull/64
+        (pkgs.fetchpatch {
+          url = "https://github.com/expipiplus1/update-nix-fetchgit/pull/64/commits/bc28c8b26c38093aa950574802012c0cd8447ce8.patch";
```

This is dangerous: From my experience you should use the url without referencing the pull request because that link will 404 e.g. when someone rebases the PR and the commit disappears.
```suggestion
          url = "https://github.com/expipiplus1/update-nix-fetchgit/commit/bc28c8b26c38093aa950574802012c0cd8447ce8.patch";
```